build/build-sdk-images/restapi/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN wget -q https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz && \
 
 RUN echo '§' && apt-get -qy update
 
-RUN apt-get install -qq -y openjdk-11-jre > /dev/null
+RUN apt-get install -qq -y openjdk-17-jre > /dev/null
```

Comment:
I know it's just one line, but is this related to the cmake changes or should it be a separate review?

Reply:
Good call out - So it has be done with the upgrade to bookworm at the same time, since there is no `openjdk-11-jre` on bookworm - so everything fails.